### DISCUSSION TITLE
"Pro" tips so you don't fall into the same trap as I did after DSM update

### DISCUSSION THREAD

**Discussion by met152** [007revad/Synology_HDD_db#165] [General]
I have a 920+ with two SSDs configured as volume2 with the help of this script.

Today I updated DSM for the first time since configuring this volume. 

Here's what I learned.  These are probably obvious to the power user, but this user failed. 

1. Make sure your Task Scheduler output logs are set to your primary volume, not your SSD volume.
2. Don't store your Synology_HDD_db script on your SSD volume, otherwise DSM won't be able to find it to run the script :(

**Comment by 007revad** [comment]
I have the scheduled task set to send me an email each time the scripts runs. It's much easier than trying to remember where I set task scheduler to save it's logs.

I might actually add a warning in the script if it's running off an M.2 drive.

**Comment by 007revad** [comment]
I've added a warning in script that warns if script is running off an M.2 drive.
